source/renderer/app/api/api.js

```diff
@@ -1221,12 +1228,16 @@ export default class AdaApi {
     Logger.debug('AdaApi::updateWallet called', {
       parameters: filterLogData(request),
     });
-    const { walletId, name } = request;
+    const { walletId, name, isLegacy } = request;
     try {
-      const wallet: AdaWallet = await updateWallet(this.config, {
-        walletId,
-        name,
-      });
+      let wallet: AdaWallet;
+      if (isLegacy) {
+        // @TODO - response is faked to enable UI. Uncomment once endpoint is available
+        // wallet = await updateByronWallet(this.config, { walletId, name });
+        wallet = await updateWallet(this.config, { walletId, name });
```

Comment:
> @yakovkaravelov replace with Byron related method

Can you clarify what is the meaning of this comment?
`updateByronWallet` method is already there inside comment.
Once you reply to first comment I added(comment or uncomment), this could be handled after that.

Reply:
@yakovkaravelov once you comment out `updateByronWallet` remove `wallet = await updateWallet(this.config, { walletId, name });` that is answer